ledger/src/blockstore_processor.rs

```diff
@@ -534,10 +535,11 @@ pub fn confirm_slot(
     timing: &mut ConfirmationTiming,
     progress: &mut ConfirmationProgress,
     skip_verification: bool,
+    skip_poh_verify: bool,
```

Comment:
This will be false for replaying blocks from blockstore on startup in `confirm_full_slot`, but true when we want asynchronous PoH verify in ReplayStage